netbout-web/src/main/js/bout.js

```diff
@@ -177,7 +177,13 @@ function scrollOrLoad(amountRecords) {
     }
   }
 }
-
+function checkUpload() {
```

Comment:
@kujtimiihoxha can you think of more descriptive name? E.g. `updateUploadButtonState` or `checkIfUploadShouldBeDisabled`.